products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,67 @@
+# Testing Mozilla.org
+
+Mozilla.org is highly visible because the site houses the basic info of all Mozilla produced products, conveys Mozilla’s mission, vision and values it stands for. Additionally, it promotes initiatives and campaigns in time of these events. As a result, new pages are created. New content is added to existing pages. New layout is introduced. The localized versions reach 60% of the Mozilla users globally. It is very important that, not only the main pages are localized, they are thoroughly tested before they are launched on production.  
+
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/{local code}/{project name}/{sub pages}
+* Pontoon: https://pontoon.mozilla.org/(local code)/mozillaorg/{project name}/{sub pages}
+* Locamotion: https://mozilla.locamotion.org/{locale code}/mozilla_lang/{project name}/{sub pages}
+* Web dashboard: https://l10n.mozilla-community.org/webdashboard/{local code}. Visit this page on a regular basis to check localization progress, pending work, deadline, and errors that were introduced during translation. 
+
+We highly advise you to ask other community members to conduct peer review not only on Pontoon or Locamotion, but on staging While not all the languages are required for certain projects, each community can opt in the projects at a later time.
+
+## What to test:
+
+### Pre-l10n test
+* Have your [glossary]((https://transvision.mozfr.org/) available as a reference, select mozilla.org as Repository, your language as your Target Locale.
+* Associate the page with a product or site if it is localized. (e.g.: Firefox, Test Pilot).
+* Have the matching US page up as reference, though some strings may not be identical due to A/B testing
+* Have the project you just localized available for editing (Pontoon or Locamotion)
+
+## Linguistic Testing
```

Comment:
If these titles are supposed to be at the same level of "pre-l10n tests", and below "What to test:", they should be 3rd level.

Other documents don't use Title case for titles, I would keep doing that for consistency.